projects/plugins/jetpack/extensions/blocks/videopress/editor.js

```diff
@@ -114,6 +138,17 @@ const addVideoPressSupport = ( settings, name ) => {
 			`jetpack/videopress-with-has-warning-is-interactive-class-names`,
 			withHasWarningIsInteractiveClassNames( `core/video` )
 		);
+	} else if ( available ) {
+		// If VideoPress is available, we update the block description and example with VideoPress-specific content.
+		settings.description = __(
+			'Embed a video from your media library or upload a new one with VideoPress.',
+			'jetpack'
+		);
+		settings.example.attributes = {
+			caption: __( 'Upload or embed videos with VideoPress', 'jetpack' ),
```

Comment:
Since we have a description right below the example, do we need the caption? Maybe it would be enough if the image showed a Play button on top of a blurry image, so folks understand it's a player?

Reply:
The idea was to emphasize on "VideoPress" so people can know we're actually doing something and it's the right block to use, but this part is so hidden that I don't really know what would be best tbh; I'm okay with removing the caption though, it doesn't add much information while requiring a new translation, so I'll update the PR 👍